app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -97,6 +100,18 @@ protected void onCreate(final Bundle savedInstanceState) {
             LOGGER.info("Bluetooth handler not started, most likely Bluetooth is not enabled");
             this.enableBluetooth(false);
         }
+        if (intent.getSerializableExtra(ConstantKeywords.SUCCESSFUL_BLUETOOTH) != null) {
+
+            final boolean successfulBluetooth = (boolean) intent.getSerializableExtra(ConstantKeywords.SUCCESSFUL_BLUETOOTH);
+
+            if (successfulBluetooth) {
```

Comment:
Oke, wierd 😕 can you replace the argument of that method (currently an `Intent`) with the result of `intent.getSerializableExtra(ConstantKeywords.SUCCESSFUL_BLUETOOTH)`